docs/pages/layouts/shell-layout.md

```diff
@@ -188,8 +188,8 @@ The container visibility can be toggled with the `aria-hidden` attribute.
     </div>
     <div class="fd-shell__overlay fd-overlay fd-overlay--alert" aria-hidden="false">
         <div class="fd-alert fd-alert--warning fd-alert--dismissible" role="alert" id="4Nolz351">
-          <button class="fd-alert__close" aria-controls="4Nolz351" aria-label="Close"></button>
-          Lorem ipsum dolor sit amet, consectetur adipisicing elit
+          <button class="fd-button fd-button--light fd-button--compact fd-alert__close" aria-controls="4Nolz351" aria-label="Close"></button>
```

Comment:
We need to keep the fd-alert__close class to keep it self contained.

Reply:
It is there actually, but we use the scss from `fd-button`.